### PR TITLE
[FIX] Delete translations of refreshed mail template contents

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: addons/purchase/migrations/10.0.1.2/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate
+def migrate(env, version):
+    """ Delete the translations of the email templates that are being
+    overwritten in the noupdate_changes.xml """
+    openupgrade.delete_record_translations(
+        env.cr, 'purchase', [
+            'module_install_notification',
```
Comment (pedrobaeza) [inline addons/purchase/migrations/10.0.1.2/pre-migration.py]: This one is a "fake" template and it doesn't need to be reset.

Comment (StefanRijnhart) [reply addons/purchase/migrations/10.0.1.2/pre-migration.py]: Yes I know, but it is included in the noupdate_data.xml so I'm just including it here as well for consistency's sake.

Comment (pedrobaeza) [reply addons/purchase/migrations/10.0.1.2/pre-migration.py]: OK, as you prefer.

Comment (StefanRijnhart) [reply addons/purchase/migrations/10.0.1.2/pre-migration.py]: On second thought, I removed both.

File: addons/purchase/migrations/10.0.1.2/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# coding: utf-8
+# Copyright 2018 Opener B.V. <stefan@opener.amsterdam>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate
```
Comment (pedrobaeza) [inline addons/purchase/migrations/10.0.1.2/pre-migration.py]: You have forgotten here `()`

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**